packages/leavittbook/src/components/leavitt-person-company-select-demo.ts

```diff
@@ -0,0 +1,158 @@
+        ></leavitt-person-company-select>
+      </div>
+    `;
+  }
+
+  #shapedStory() {
+    return html`
+      <div>
+        <h1>Shaped</h1>
+        <p>Shaped select styles</p>
+        <leavitt-person-company-select
+          label="Leavitt person or company"
+          shaped
+          .apiService=${api3UserService}
+          validationMessage="Select a person or company"
+        ></leavitt-person-company-select>
+      </div>
+    `;
+  }
+
+  #shallowStory() {
+    return html`
+      <div>
+        <h1>Shallow</h1>
+        <p>Shallow select styles</p>
+        <leavitt-person-company-select
+          label="Leavitt person or company"
+          shallow
+          .apiService=${api3UserService}
+          validationMessage="Select a person or company"
+        ></leavitt-person-company-select>
+      </div>
+    `;
+  }
+
+  #preselectedStory() {
+    return html`
+      <div>
+        <h1>Pre-selected</h1>
+        <p>Pre-selected styles</p>
+        <leavitt-person-company-select
+          label="Leavitt person or company"
+          .apiService=${api3UserService}
+          .selected=${{
+            Name: 'Leavitt Group Enterprises',
+            type: 'Company',
+          } as any}
+          validationMessage="Select a person or company"
+        ></leavitt-person-company-select>
+      </div>
+    `;
+  }
+
+  #disabledPreselectedStory() {
+    return html`
+      <div>
+        <h1>Shallow</h1>
+        <p>Shallow select styles</p>
+        <leavitt-person-company-select
+          label="Leavitt person or company"
+          disabled
+          .selected=${{
+            Name: 'Leavitt Group Enterprises',
+            type: 'Company',
+          } as any}
+          .apiService=${api3UserService}
+          validationMessage="Select a person or company"
+        ></leavitt-person-company-select>
+      </div>
+    `;
+  }
+
+  render() {
+    return html`
+      <story-header name="Leavitt person company select" tagName="leavitt-person-company-select" klass="LeavittPersonCompanySelectElement"></story-header>
+      <titanium-card>
+        ${this.#defaultStory()}
+        <code-block .snippet=${this.#defaultStory()}> </code-block>
+      </titanium-card>
+      <titanium-card>
+        ${this.#requiredStory()}
+        <code-block .snippet=${this.#requiredStory()}> </code-block>
+      </titanium-card>
+      <titanium-card>
+        ${this.#shapedStory()}
+        <code-block .snippet=${this.#shapedStory()}> </code-block>
+      </titanium-card>
+      <titanium-card>
+        ${this.#shallowStory()}
+        <code-block .snippet=${this.#shallowStory()}> </code-block>
+      </titanium-card>
+      <titanium-card>
+        ${this.#preselectedStory()}
+        <code-block .snippet=${this.#preselectedStory()}> </code-block>
+      </titanium-card>
+      <titanium-card>
+        ${this.#disabledPreselectedStory()}
+        <code-block .snippet=${this.#disabledPreselectedStory()}> </code-block>
+      </titanium-card>
```

Comment:
Need focus() and reset() demos

Reply:
Specifically for the disabled input? I can do that, but it feels wrong to reset that input